.github/workflows/build_and_scan_rocks.yaml

```diff
@@ -5,16 +5,25 @@ on:
   #schedule:
   # every day at 1:12AM UTC
   #- cron: '12 1 * * *'
+    secrets:
+      JIRA_URL:
```

Comment:
I realised that jira url name might clash with secret name for gh jira issue sync name

Reply:
Let's come up with something else.

Reply:
Done.